tests/tests.mk

```diff
@@ -19,6 +19,14 @@ doc.test: init | $(JUNIT_REPORT)
 		-istio.test.tag=$(TAG) \
 		2>&1 | tee >($(JUNIT_REPORT) > $(JUNIT_OUT))
 
+doc.multicluster.test: init | $(JUNIT_REPORT)
+	${GO} test ${REPO_ROOT}/tests/multicluster/setup/... \
+	-v -timeout=${TIMEOUT} -count=1 -p=1 \
+	-istio.test.hub=$(HUB) \
+	-istio.test.tag=$(TAG) \
+	-istio.test.kube.config=${DOCTEST_KUBECONFIG} \
+	2>&1 | tee >($(JUNIT_REPORT) > $(JUNIT_OUT))
```

Comment:
Why do we need a special rule for multicluster? I would think `make doc.test` should support all doc tests, multicluster or not.

Reply:
I tend to agree ... could we just treat `multicluster` as a separate profile for now? I suspect if we do that, this PR gets a lot simpler.

Reply:
> could we just treat multicluster as a separate profile

It's not a different "profile", but it is a different "setup", so that should fit cleanly into the current doc.test design. See https://github.com/istio/istio.io/pull/8160#discussion_r490428659 and https://github.com/istio/istio.io/pull/8160#discussion_r490430352.

Reply:
> separate directory under setup

Yes. https://github.com/istio/istio.io/pull/8160#discussion_r490440174

Reply:
So then the question will be that we either need to add another test to cover the new directory, as the existing tests call the other "setups". Or we go back to a single test that includes all the "setups".  The issue with the later is that the time to run the single test sill be roughly the sum of the four tests, and if a single test fails in one of the "setups", we would need to run all the "setups" again instead of the single failing "setup".

Reply:
Inadvertently deleted my earlier comment.

If we put the tests in another "setup" we will need to create a new test to pick up the new "setup". The negative to this is we are increasing the number of tests, since we would have one per "setup". The positive to having a test per "setup" is we have a set of shorter tests instead of combining them into a longer running test, roughly the sum of the times of each individual "setup" test. Also, as separate "setup" test, one only needs to run that "setup" again, instead of all the tests again.

Reply:
I agree with @frankbu . This is a different setup. We have one more condition here - we need at least 2 clusters (although in configuration we have 3)